src/components/TiktokScroll/index.js

```diff
@@ -22,6 +22,14 @@ class TiktokScroll extends React.Component {
     });
   }
 
+  componentDidMount() {
+    if (this.props.contentInsetAdjustmentBehavior === 'never') {
+      setTimeout(() => {
+        this.scrollToTop();
+      }, 200);
+    }
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `componentDidMount` lifecycle method is used to call the `scrollToTop` function after a delay of 200ms if the `contentInsetAdjustmentBehavior` prop is set to 'never'. However, there's no check for whether the component is still mounted when the timeout completes. This could potentially lead to a memory leak if the component unmounts before the timeout finishes. To avoid this, you should clear the timeout in the `componentWillUnmount` lifecycle method.
```diff
+  componentDidMount() {
+    if (this.props.contentInsetAdjustmentBehavior === 'never') {
+      this.timeoutId = setTimeout(() => {
+        this.scrollToTop();
+      }, 200);
+    }
+  }
+
+  componentWillUnmount() {
+    if (this.timeoutId) {
+      clearTimeout(this.timeoutId);
+    }
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->